settings.py

```diff
@@ -453,12 +472,12 @@
 
     SOCIAL_AUTH_PIPELINE = ('seqr.utils.social_auth_pipeline.validate_anvil_registration',) + \
                            SOCIAL_AUTH_PIPELINE_BASE + \
-                           ('social_core.pipeline.user.get_username',
-                            'social_core.pipeline.user.create_user') + \
-                           SOCIAL_AUTH_PIPELINE_ASSOCIATE_USER + \
-                           ('social_core.pipeline.social_auth.load_extra_data',
-                            'social_core.pipeline.user.user_details') + \
+                           SOCIAL_AUTH_PIPELINE_CLOUD_BASE + \
                            SOCIAL_AUTH_PIPELINE_LOG
+elif SOCIAL_AUTH_AZUREAD_V2_TENANT_OAUTH2_KEY:
+    SOCIAL_AUTH_PIPELINE = SOCIAL_AUTH_PIPELINE_BASE + \
```

Comment:
So this auth approach allows anyone in the world with a valid azure account to log into your seqr instance, which I don't really feel comfortable making the defautl behavior for all seqr Azure instances.  Currently we have 2 ways of validating users when google auth is enabled: 
1) If terra/anvil integration is enabled, confirm that the google account is registered with anvil
2) If the terra integration is not enabled, require the account to have been explicitly added to at least one seqr project (the `SOCIAL_AUTH_PIPELINE_USER_EXIST` step)

I think approach 2 (pipeline defined in the "else" block) would be a better and more secure default behavior

Reply:
no, actually the `AzureADV2TenantOAuth2` provider is tenant-specific (see `SOCIAL_AUTH_AZUREAD_V2_TENANT_OAUTH2_TENANT_ID`), so the approach is to create a SEQR instance-specific tenant (which is what we've done with our various test instances) - only users defined directly by that tenant or explicitly guested in from a federated auth provider have access...